v_2_0_0/master_template.go

```diff
@@ -10,167 +10,6 @@ users:
        - "{{ $user.PublicKey }}"
 {{end}}
 write_files:
-- path: /srv/calico-ipip-pinger-sa.yaml
```

Comment:
We should not modify versions that are already established. I had the hope Joe was the exception because of monitoring. When we want to patch a specific version we should create a new version based of it. I think in this case it might be enough to just put it into v3.

Reply:
Agree to bump versions. But in general, this is important bugfix release. We need to ship Calico 2.6.4 to existing clusters.

If i still want to  do changes in v2_x_x. Can you please describe steps i need to do (i think you're the only SME about versions at the moment)?

1) `cp -r v2_0_0 v2_0_1`
2) Apply my changes to v2_0_1
3) ...

Reply:
How do we want to ship this to existing clusters? Are you updating all of them manually somehow? How did we do this before? We never updated anything automatically. Being able to do so is still WIP. AWS is not reconciling and KVM has updates disabled. So even if we modify existing versions we do not really apply them. You would need to roll everything manually somehow when this is so important. 

I am against modifying existing versions that are already in use. Don't know how often I have to say this still. I am wondering why everything is now so important that we have to bypass the process we want to establish. 

You can create a new version but then customers have to create new clusters to get this very important bugfix.

Reply:
I did not know that we disabled automatic updates in kvm-operator (and in aws-operator), so i expected that existing versions will be rolled out once vm recreated.

Can we spec our upgrade/versioning strategy at least in common sentences? It's total miscommunication on such important topic. And looks like it's our common fault.

Can we have a call on that? may be within the sig syncup?

cc: @calvix @teemow @puja108

Reply:
I'd like to split the PR in to two: one with with v_2_0_1; one with update to v_3_0_0.
I agree with Tim. we should never change existing version. Even if it's blank line removal. If we want to roll patches automatically we can implement that in one of our services like cluster-service or kubernetesd.